src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-server.h

```diff
@@ -164,7 +166,7 @@ class Delegate
      *   - CHIP_NO_ERROR on success
      *   - An error if the session is invalid or the candidates cannot be processed
      */
-    virtual CHIP_ERROR HandleProvideICECandidates(uint16_t sessionId, const std::vector<std::string> & candidates) = 0;
+    virtual CHIP_ERROR HandleProvideICECandidates(uint16_t sessionId, const std::vector<ICECandidateStruct> & candidates) = 0;
```

Comment:
This needs to document the memory situation with the ICECandidateStruct fields.  And fix the @param documentation for the "candidates" arg in general, because now it's wrong.

In particular, callee must not keep referencing passed-in buffers after the call returns, right?

Reply:
param doc fixed with a Note about the requirement that the contents of the struct need to be copied over for later use.